dnf-docker-test/features/steps/rpm_utils.py

```diff
@@ -47,17 +48,36 @@ def hdr2nevra(hdr):
     """
     return hdr["nevra"].decode() if hdr else None
 
-def find_pkg(pkgs, name):
+def find_pkg(pkgs, pkg):
     """
     :param list(rpm.hdr) pkgs: List of RPM Headers
-    :param str name: Package name to find
+    :param str pkg: Package to find
     :return: First found RPM header
     :rtype: rpm.hdr or None
     """
-    try:
-        return next(pkg for pkg in pkgs if pkg.name == name.encode())
-    except StopIteration:
-        return None
+    epoch = version = release = None
+    if "/" in pkg:
+        name, evr = pkg.split("/")
+    else:
+        name = pkg
+        if ":" in evr:
+            epoch = evr.split(":", 1)[0]
+            evr = evr[len(epoch) + 1:]
+        if "-" in evr:
+            version, release = evr.split("-")
+        else:
+            version = evr
+
```

Comment:
The block doing the EVR parsing starting line 63 doesn't belong to this "else" branch bug the "if" branch above it.